smartsim/_core/_cli/validate.py

```diff
@@ -171,6 +171,20 @@ def _make_managed_local_orc(
         exp.stop(orc)
 
 
+@contextmanager
+def _disable_telmon() -> t.Generator[None, None, None]:
+    """Ensure the telemetry monitor is disabled during a test and the environment
+    is left in correct state after completion"""
+    orig = os.environ.get("SMART_TELEMETRY_ENABLED", None)
```

Comment:
Side note: ideally I would like to avoid querying env in this module entirely and leave it entirely in `smartsim._core.config`.

I tried playing around with it, but couldn't come up with a non-trivial way not querying the env. At this point I am willing to over look for the name of faster merge time, so don't spend too much time looking into this suggestion!

Reply:
fixed key name fail.